src/components/TableBrowser/NetworkInfoPanel.tsx

```diff
@@ -107,102 +139,76 @@ export default function NetworkInfoPanel(props: {
               sx={{ ml: 1, mr: 4, fontSize: 14, color: 'gray' }}
               variant="subtitle1"
             >
-              {`Owner: ${networkInfo?.owner}`}
+              {`Owner: ${networkInfo.owner}`}
             </Typography>
             <Typography
               sx={{ mr: 4, fontSize: 14, color: 'gray' }}
               variant="subtitle1"
             >
-              {`Created: ${dateFormatter(networkInfo?.creationTime ?? '')}`}
+              {`Created: ${dateFormatter(networkInfo.creationTime)}`}
```

Comment:
The old code seems to be safer. Why do we removed the null checkers in these lines?